frontend/src/main/components/Reports/ReportLineTable.js

```diff
@@ -14,6 +14,7 @@ export default function ReportLineTable({ reportLines }) {
         {
             Header: 'Total Wealth',
             accessor: 'totalWealth',
+            Cell: ({ value }) => `$${value}`
```

Comment:
Not sure why changes to this file are in this PR.  I feel like this is going to become a theme...